cmd/xl-v1-object.go

```diff
@@ -156,6 +156,19 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 		}
 	}
 
+	// Validate if all the parts exist before proceeding to read from the disk.
+	for _, part := range xlMeta.Parts {
+		// Pick the first disk from online disks and validate if we have all the parts.
+		_, err = onlineDisks[0].StatFile(bucket, path.Join(object, part.Name))
```

Comment:
onlineDisks[0] ? won't this cause a permanent problem when the first disk is unaccessible but we have quorum ?